src/Dashboard.js

```diff
@@ -136,10 +136,7 @@ function Dashboard() {
         ))}
       </h1>{" "}
       <Container fluid>
-        <Row
```

Comment:
@ajinsunny I think it was incorrect to take out the padding. Padding must be kept.